src/app/globals.css

```diff
@@ -23,11 +23,127 @@
 body {
   background: var(--background);
   color: var(--foreground);
-  font-family: "Mulish", Arial, Helvetica, sans-serif;
+  font-family: "Open Sans", Arial, Helvetica, sans-serif;
 }
 
-.mulish-uniquifier{
-  font-family: "Mulish", sans-serif;
+.open-sans-ifdm {
+  font-family: "Open Sans", sans-serif;
```

Comment:
You could probably keep the rest of these, but remove this line referencing Open Sans, as that's already being imported.